### PR TITLE
mgr/dashboard: Add Polish language

### PR DESCRIPTION
**Pull request by dziomdziorae**
Fixes: https://tracker.ceph.com/issues/38236

Signed-off-by: Elzbieta Dziomdziora <elzbieta.dziomdziora@ts.fujitsu.com>

- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/src/app/shared/components/language-selector/supported-languages.enum.ts [hunk 1]
```diff
@@ -6,5 +6,6 @@ export enum SupportedLanguages {
   'id-ID' = 'Bahasa Indonesia',
   'pt-PT' = 'Português',
   'es-ES' = 'Español',
-  'zh-CN' = '中文'
+  'zh-CN' = '中文', 
+  'pl-PL' = 'Polish'
```
Comment (tchaikov) [inline src/pybind/mgr/dashboard/frontend/src/app/shared/components/language-selector/supported-languages.enum.ts]:
```suggestion
  'pl-PL ' = 'Polski'
```

i think the comment from @tspmelo https://github.com/ceph/ceph/pull/26907#discussion_r264641671 still applies

**Comment by tchaikov** [comment]
and the comment from @LenzGr also still applies. see https://github.com/ceph/ceph/pull/26907#issuecomment-471957967

**Comment by tchaikov** [comment]
and https://github.com/ceph/ceph/pull/26907#pullrequestreview-213343388

**Comment by tchaikov** [comment]
see also #26907

**Comment by votdev** [comment]
Please modify the commit message and the PR text to
```
mgr/dashboard: Add Polish language

Fixes: https://tracker.ceph.com/issues/38236

Signed-off-by: Elzbieta Dziomdziora <elzbieta.dziomdziora@ts.fujitsu.com>
```
See https://github.com/ceph/ceph/pull/26907#issuecomment-471957967.

**Comment by Exotelis** [comment]
Hi,

just opened https://github.com/ceph/ceph/pull/27247 with the language file included and already tested ;)

**Comment by LenzGr** [comment]
Closing this one in favor of #27247

**Pull request closed without merging**